src/main/java/io/github/cottonmc/cotton_scripting/mixin/TagContainerMixin.java

```diff
@@ -0,0 +1,23 @@
+package io.github.cottonmc.cotton_scripting.mixin;
+
+import io.github.cottonmc.cotton_scripting.TagContainerAccessor;
+import net.minecraft.tag.Tag;
+import net.minecraft.tag.TagContainer;
+import net.minecraft.util.Identifier;
+import org.spongepowered.asm.mixin.Implements;
+import org.spongepowered.asm.mixin.Interface;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+
+import java.util.Map;
+
+@Mixin(TagContainer.class)
+@Implements(@Interface(iface = TagContainerAccessor.class,prefix = "tagContainerAccessor$",remap = Interface.Remap.ALL))
```

Comment:
If this is something internal-only, and a dev working on an addon is never gonna touch this, then no need to use `@Implements`; we can just have it implement in the normal way and prefix the method in `TagContainerAccessor`.

Reply:
I'm always attempting to play it safe.

Reply:
that's fair